src/css/diario/etapas/etapas.css

```diff
@@ -0,0 +1,32 @@
+td{
+  padding: 15px 80px;
+}
```

Comment:
TODO o css externo do trabalho deve estarnum arquivo chamado "estilo.css" de caminho : src/css/estilo.css
Por isso quando for estilizar uma estrutura com o css externo utiliza uma referencia mais especifica possível. Por exemplo,nesse caso você estaria estilizando todo os "td's" do trabalho inteiro. Então tem que usar id's e classes mais especificas como: #td-tabela-acervo-blabla,para nao dar conflito.